guides/example/page_functionality_wix_sdk.md

```diff
@@ -16,33 +16,57 @@ You add permissions to your app in the [Developers Center](https://dev.wix.com/)
 
 After setting up permissions, our app can get a list of products from the site's store.
 
-We start by importing and installing the following:
+First, install the dependencies we'll need for working with the SDKs:
+
+```
+npm install @wix/sdk-react @wix/dashboard-react @wix/stores react-query
+```
+
+In out code, we start by wrapping our exported page component with the [`withDashboard`](TODO: link here) HOC. This will allow us to later use the [`useWixModules`](https://dev.wix.com/docs/sdk/api-reference/sdk-react/hooks#usewixmodules) hook to call Wix APIs.
```

Comment:
typo
```suggestion
In our code, we start by wrapping our exported page component with the [`withDashboard`](TODO: link here) HOC. This will allow us to later use the [`useWixModules`](https://dev.wix.com/docs/sdk/api-reference/sdk-react/hooks#usewixmodules) hook to call Wix APIs.
```